packages/mobile/android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         targetSdkVersion = 31
         castFrameworkVersion = "21.0.0"
         ndkVersion = "21.4.7075529"
-        ffmpegKitPackage = "https-lts"
+        ffmpegKitPackage = "https-gpl-lts"
```

Comment:
i guess this too? need GPL version for android?

Reply:
Yeah so we needed to use the libx264 encoder to produce the properly formatted video with H.264/AAC, which is included in the GPL version but not the regular one. then i also needed to add `file://` (for some reason it was working without this in iOS, it's like a small print thing in the react-native-share docs)

Reply:
Awesome finds, wow